src/pages/Home.js

```diff
@@ -1,71 +1,164 @@
-    <FooterNavigation />
+      <Headline Today={Today}> This is the today page. </Headline>
+      <EditButtonWrapper>
+        {habits.length !== 0 &&
+          (editMode === false ? (
+            <IconButton onClick={() => setEditMode(true)}>
+              <img src={editIcon} alt="" height="20px" />
+            </IconButton>
+          ) : (
+            <IconButton align="right" onClick={() => setEditMode(false)}>
+              <img src={returnIcon} alt="" height="20px" />
+            </IconButton>
+          ))}
+      </EditButtonWrapper>
+
+      <div>
+        {habits.map((habit) => {
+          if (habit.häufigkeit == "täglich") {
+            return (
+              <>
+                {checkbox(habit.ziel)}
+
+                {editMode && (
+                  <>
+                    <IconButton onClick={() => handleDeleteClick(habit.id)}>
+                      <img src={deleteIcon} alt="löschen" height="16" />
+                    </IconButton>
+                    <IconButton
+                      onClick={() => {
+                        setShowsEditModal(true);
+                        onSetHabitToEdit(habit);
+                      }}
+                    >
+                      <img src={editIcon} alt="bearbeiten" height="16" />
+                    </IconButton>
+                  </>
+                )}
+              </>
+            );
+          }
+        })}
+      </div>
+
+      {showsEditModal && (
+        <HabitWrapper>
+          <HabitForm
+            setShowsEditModal={setShowsEditModal}
+            habitToEdit={habitToEdit}
+            onEditHabit={onEditHabit}
+          />
+        </HabitWrapper>
+      )}
+
+      <NavLink to="/tracker" className="link">
+        <ButtonWrapper>
+          <Button>zum Tracker</Button>
+        </ButtonWrapper>
+      </NavLink>
+      <FooterNavigation />
     </>
   );
 }
 
 const Button = styled.button`
-    background-color: transparent;
-    border-radius: 100vw;
-    cursor: pointer;
-    font-size: 1rem;
-    padding: 0.5rem;
-    text-decoration: none;
-`
+  background-color: transparent;
+  border-radius: 100vw;
+  cursor: pointer;
+  font-size: 1rem;
+  padding: 0.5rem;
+  text-decoration: none;
+`;
 
 const ButtonWrapper = styled.section`
-    align-items: center;
-    display: flex;
-    margin-left: 8.5rem;
-    margin-top: 430px;
-`
+  align-items: center;
+  display: flex;
+`;
+
+const EditButtonWrapper = styled.div`
+  align-items: end;
+  background-color: transparent;
+  border: none;
+  display: flex;
+  justify-content: end;
+`;
 
 const Headline = styled.h1`
-    display: flex;
-    align-items: center;
-    font-family: 'Roboto'
-`
\ No newline at end of file
+  display: flex;
+  align-items: center;
+  font-family: "Roboto";
+`;
+
+const HabitWrapper = styled.div`
+  background-color: rgba(255, 255, 255, 0.7);
+  backdrop-filter: blur(10px);
+  display: block;
+  position: absolute;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
```

Comment:
is this needed? What are you wanting to create here?

Reply:
It is, I created a modal and I want it to be placed in the middle of the screen.

Reply:
does `margin: 0, auto;` work?

https://stackoverflow.com/questions/3170772/what-does-auto-do-in-margin0-auto

Reply:
for some reason it doesn't work here. the modal gets placed into the bottom left corner.